src/library/bpmdelegate.cpp

```diff
@@ -18,10 +19,10 @@ class BpmEditorCreator : public QItemEditorCreatorBase {
     }
 
     QWidget* createWidget(QWidget* parent) const override {
-        QDoubleSpinBox* pBpmSpinbox = new QDoubleSpinBox(parent);
+        WDoubleSpinBox* pBpmSpinbox = new WDoubleSpinBox(parent);
         pBpmSpinbox->setFrame(false);
         pBpmSpinbox->setMinimum(0);
-        pBpmSpinbox->setMaximum(1000);
+        pBpmSpinbox->setMaximum(99999);
         pBpmSpinbox->setSingleStep(1e-3);
         pBpmSpinbox->setDecimals(8);
```

Comment:
Not related to this PR but these settings feel overly precise, would you know if that something that was requested at some point or just arbitrary value that could be reviewed for a better UX?

Reply:
Yeah, 8 decimals look scary. I think there are two reason:
* for some tracks it seems it's not possible to match the BPM with `beats_adjust_slower/_faster` (`kBpmAdjustStep` is 0.01), the spinbox allows finer adjusments
* have a chance to discover that the BPM display is rounded when there are more than let's say 4 decimals